recipes/ckanbatch-deploy.rb

```diff
@@ -83,6 +83,25 @@
         })
         action [:create]
     end
+    systemd_unit "ckan-worker2.service" do
```

Comment:
Do we have the ram headroom?

Reply:
Should do. Also, the pending changes to chunk processing should reduce peak memory usage.